src/Commands/Lock/EnableCommand.php

```diff
@@ -7,29 +7,29 @@
 use Pantheon\Terminus\Site\SiteAwareTrait;
 
 /**
- * Class AddCommand
+ * Class EnableCommand
  * @package Pantheon\Terminus\Commands\Lock
  */
-class AddCommand extends TerminusCommand implements SiteAwareInterface
+class EnableCommand extends TerminusCommand implements SiteAwareInterface
 {
     use SiteAwareTrait;
 
     /**
-     * Lock an environment with a username and password
-     * Note: The username and password are stored in plaintext format on the server.
+     * Enables HTTP Basic Auth on the environment.
+     * Note: Basic Auth username and password are stored in plaintext.
```

Comment:
A (certainly not-comprehensive) look about the internet found nowhere where "Basic Auth" is a proper noun. Indeed, almost everywhere it was expanded to "basic authentication".